pkg/network/infraconfigurators/masquerade.go

```diff
@@ -376,6 +377,18 @@ func (b *MasqueradePodNetworkConfigurator) createNatRulesUsingNftables(proto ipt
 		return err
 	}
 
+	// Istio excludes port 22 from proxying, we need to forward this prot explicitly
+	// Can be removed when Istio provides a with a way to not exclude the port 22 from proxying.
+	// https://issues.redhat.com/browse/OSSM-782
+	if istio.ProxyInjectionEnabled(b.vmi) {
```

Comment:
If we have a specific port list for the vm and port 22 is not there, I believe we don't need to forward it to the vm. I think the ssh port exclusion should be added as `or` to the `KUBEVIRT_PREINBOUND` existing code adding the rules ([no ports](https://github.com/kubevirt/kubevirt/pull/6923/commits/b37d377a54a44b57a81a348086c1c0ba16ab8750#diff-5cc59c701f44b3d4b9d3f0e79b5098fca04ac0c4f1218eb74c6d3cbe36da6649R407) and [with ports](https://github.com/kubevirt/kubevirt/pull/6923/commits/b37d377a54a44b57a81a348086c1c0ba16ab8750#diff-5cc59c701f44b3d4b9d3f0e79b5098fca04ac0c4f1218eb74c6d3cbe36da6649R440).

Reply:
Done